tests/yast2_gui/yast2_control_center.pm

```diff
@@ -33,6 +33,7 @@ sub run() {
         send_key 'backspace';
     }
 
+
```

Comment:
okay, empty line got removed

Please check reference test: http://e13.suse.de/tests/2312 and /tests/2314
Since there is a local server issue, it is hard to get all tests down without failure. But I am sure that that all tests after Windows Domain Membership are okay..